### ISSUE TITLE
Use with Chromatic

### ISSUE THREAD

**Issue by rangerscience** [YusukeIwaki/playwright-ruby-client#320]
### Simple description about the feature

Chromatic has a very simple integration with Playwright: https://www.chromatic.com/docs/playwright/

Replace:
```
import { test, expect } from "@playwright/test"; 
```

with:

```
import { test, expect } from "@chromatic-com/playwright"; 
```

Any pointers for where to look in this gem to allow or make this change? 

(I think there might be another way using the JS module resolution stuff, but I'm still new to all that...)

### Usecase / Motivation

Allows you to use this gem *and* Chromatic.

**Comment by YusukeIwaki** [comment]
Hi, thank you for your suggestion.

playwright-ruby-client is browser-automation tool,
and visual regression testing feature would be provided by a plugin for testing library (rspec-xxxx, minitest-xxxx).

**Comment by rangerscience** [comment]
Yes, Chromatic provides visual regression testing, and does so as a plugin/testing library, by replacing the `"@playwright/test"` with `"@chromatic-com/playwright"`. Reading Chromatic's docs, it does this because it uses the browser automation to capture HTML as the visual test, rather than a screenshot image.

Let me try a different question: Where does `"@playwright/test"` get used by this gem? :)

**Comment by YusukeIwaki** [comment]
@rangerscience `@playwright/test` is just a test runner (test framework) like mocha, jest and not a browser-automation library.

This playwright-ruby-client gem provides feature for browser-automation and assertion, however no test runner (framework) features are provided.
